src/Backends/Banshee.GStreamer/Banshee.GStreamer/PlayerEngine.cs

```diff
@@ -209,9 +209,11 @@ protected override void Initialize ()
 
         public override void Dispose ()
         {
+            bp_stop (handle, true);
```

Comment:
I'm still unsure we need this one since it appears to be called from `bp_destroy`, with this line commented out I still get this log when closing the program:

```
[1 Debug 15:01:55.078] (libbanshee:player) bp_stop: setting state to GST_STATE_NULL
[1 Debug 15:01:55.084] Player state change: Playing -> Idle
[1 Debug 15:01:55.087] (libbanshee:player) bp_destroy: disposed player
[1 Debug 15:01:55.087] (libbanshee:gst) Deinitializing GStreamer
[1 Debug 15:01:55.089] Service disposed (PlayerEngine)
[1 Debug 15:01:55.098] Service disposed (SourceManager)
[1 Debug 15:01:55.101] Service disposed (DbConnection)
```